dom.bs

```diff
@@ -1554,8 +1564,16 @@ them. For instance, if the operation has already completed.
 
  <li><p>Set <var>signal</var>'s [=AbortSignal/aborted flag=].
 
- <li><p><a for=set>For each</a> <var>algorithm</var> in <var>signal</var>'s
- [=AbortSignal/abort algorithms=]: run <var>algorithm</var>.
+ <li>
+  <p><a for=set>For each</a> <var>algorithm</var> in <var>signal</var>'s
+  [=AbortSignal/abort algorithms=]:
+
+  <ol>
+   <li><p>Run <var>algorithm</var>.
+
+   <li><p>Remove <var>algorithm</var> from <var>signal</var>'s
+   <a for=AbortSignal>abort algorithms</a>.
```

Comment:
Mutating a list as you loop through it, although it's probably well-defined, can be confusing, and easy to implement correctly. I'd suggest just clearing the list after the loop.